dtfs-central-api/src/v1/routes/middleware/payload-validation/validate-post-upload-utilisation-report-payload.ts

```diff
@@ -15,15 +15,16 @@ export type PreValidationPostUploadUtilisationReportRequest = CustomExpressReque
  * @param data - The report data
  * @returns Array of validation errors
  */
-export const getReportDataErrors = (data: Record<string, string | null>[]) => {
+export const getReportDataErrors = async (data: Record<string, string | null>[]) => {
   const reportDataMapped = data.map((reportDataRow, rowIndex) => {
     return Object.keys(reportDataRow).reduce((rowData, key) => {
       // We add two to the index to get the row to account for zero-indexing and the header row
       return { ...rowData, [key]: { value: reportDataRow[key], row: rowIndex + 2 } };
     }, {} as UtilisationReportCsvRowData);
   });
 
-  return validateUtilisationReportCsvData(reportDataMapped).map(({ errorMessage, value, row }) => ({ errorMessage, value, row }));
+  const validationErrors = await validateUtilisationReportCsvData(reportDataMapped);
+  return validationErrors.map(({ errorMessage, value, row }) => ({ errorMessage, value, row }));
```

Comment:
Do we need to do this mapping, what happens if we just return `validationErrors` as is?

Reply:
We get lots of extra fields with the value being just null. Given this response is not actually used upstream but it useful for logging (e.g. we actually only expect to get errors if a user is being malicious or something has gone wrong with the journey) it is much nicer to read and easier to debug when we reduce to just the fields we expect to actually possibly have a value.